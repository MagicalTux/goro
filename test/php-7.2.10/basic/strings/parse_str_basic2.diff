     *** Testing parse_str() : non-default arg_separator.input specified ***
002- NULL
003- array(3) {
002+ array(1) {
       ["first"]=>
005-   string(4) "val1"
006-   ["second"]=>
007-   string(4) "val2"
008-   ["third"]=>
009-   string(4) "val3"
004+   string(27) "val1/second=val2/third=val3"
005+ }
006+ array(1) {
007+   ["first"]=>
008+   string(27) "val1/second=val2/third=val3"
     }
     ===DONE===
