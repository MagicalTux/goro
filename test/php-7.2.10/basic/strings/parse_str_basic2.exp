*** Testing parse_str() : non-default arg_separator.input specified ***
NULL
array(3) {
  ["first"]=>
  string(4) "val1"
  ["second"]=>
  string(4) "val2"
  ["third"]=>
  string(4) "val3"
}
===DONE===