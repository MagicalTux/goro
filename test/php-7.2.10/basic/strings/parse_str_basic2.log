
---- EXPECTED OUTPUT
*** Testing parse_str() : non-default arg_separator.input specified ***
NULL
array(3) {
  ["first"]=>
  string(4) "val1"
  ["second"]=>
  string(4) "val2"
  ["third"]=>
  string(4) "val3"
}
===DONE===
---- ACTUAL OUTPUT
*** Testing parse_str() : non-default arg_separator.input specified ***
array(1) {
  ["first"]=>
  string(27) "val1/second=val2/third=val3"
}
array(1) {
  ["first"]=>
  string(27) "val1/second=val2/third=val3"
}
===DONE===
---- FAILED
