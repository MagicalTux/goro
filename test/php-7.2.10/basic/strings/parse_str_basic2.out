*** Testing parse_str() : non-default arg_separator.input specified ***
array(1) {
  ["first"]=>
  string(27) "val1/second=val2/third=val3"
}
array(1) {
  ["first"]=>
  string(27) "val1/second=val2/third=val3"
}
===DONE===